elements/lisk-transactions/src/0_transfer.ts

```diff
@@ -120,7 +126,8 @@ export const transfer = (inputs: TransferInputs): Partial<TransactionJSON> => {
 	const transferTransaction = new TransferTransaction(
 		transactionWithSenderInfo,
 	);
-	transferTransaction.sign(passphrase, secondPassphrase);
+
+	transferTransaction.sign(networkIdentifier, passphrase, secondPassphrase);
```

Comment:
IMO the `sign` method should only take `passphrase` and `secondPassphrase` as param.
The `networkIdentifier` could be set in the constructor, for example.